setup/src/main/java/ch/minova/install/setup/BaseSetup.java

```diff
@@ -147,15 +162,19 @@ public boolean readoutSchemaCreate(final Connection con, Optional<Path> tableLib
 		} catch (final SQLException e1) {
 			System.out.println("help me SQLException");
 			e1.printStackTrace();
+			logger.logSetup(e1.getMessage());
 		} catch (final InstantiationException e1) {
 			System.out.println("help me InstantiationException");
 			e1.printStackTrace();
+			logger.logSetup(e1.getMessage());
 		} catch (final IllegalAccessException e1) {
 			System.out.println("help me IllegalAccessException");
 			e1.printStackTrace();
+			logger.logSetup(e1.getMessage());
 		} catch (final ClassNotFoundException e1) {
 			System.out.println("help me ClassNotFoundException");
 			e1.printStackTrace();
+			logger.logSetup(e1.getMessage());
```

Comment:
Bitte die Stacktraces direkt über den logger loggen und `e1.printStackTrace();` entfernen.
Falls dies im CustomLogger nicht definiert ist, kann man dort eine neue Log-Methode dazu einführen und dort notfallshalber PrintWriter direkt verwenden.

Reply:
Keine Ahnung wie viel cooler das jetzt ist, wenn man des Setup-Log anschaut, aber dort die Fehler nicht sieht.
Aber prinzipiell landen sie in einer der Dateien, das ist ja schon mal was.